pylint output

************* Module jogo
jogo.py:320:0: C0301: Line too long (106/100) (line-too-long)
jogo.py:324:0: C0301: Line too long (105/100) (line-too-long)
jogo.py:331:0: C0301: Line too long (106/100) (line-too-long)
jogo.py:336:0: C0301: Line too long (105/100) (line-too-long)
jogo.py:404:0: C0301: Line too long (114/100) (line-too-long)
jogo.py:411:0: C0301: Line too long (106/100) (line-too-long)
jogo.py:437:0: C0301: Line too long (117/100) (line-too-long)
jogo.py:482:0: C0301: Line too long (113/100) (line-too-long)
jogo.py:488:0: C0301: Line too long (107/100) (line-too-long)
jogo.py:494:0: C0301: Line too long (113/100) (line-too-long)
jogo.py:500:0: C0301: Line too long (107/100) (line-too-long)
jogo.py:1:0: C0114: Missing module docstring (missing-module-docstring)
jogo.py:3:0: E0401: Unable to import 'pygame' (import-error)
jogo.py:19:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:25:12: C0103: Variable name "BackGround" doesn't conform to snake_case naming style (invalid-name)
jogo.py:19:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:35:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:41:12: C0103: Variable name "StartScreen" doesn't conform to snake_case naming style (invalid-name)
jogo.py:35:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:51:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:71:8: C0103: Attribute name "speedX" doesn't conform to snake_case naming style (invalid-name)
jogo.py:72:8: C0103: Attribute name "speedY" doesn't conform to snake_case naming style (invalid-name)
jogo.py:57:12: C0103: Variable name "player1Img" doesn't conform to snake_case naming style (invalid-name)
jogo.py:74:4: C0116: Missing function or method docstring (missing-function-docstring)
jogo.py:51:0: R0903: Too few public methods (1/2) (too-few-public-methods)
jogo.py:79:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:100:8: C0103: Attribute name "speedX" doesn't conform to snake_case naming style (invalid-name)
jogo.py:101:8: C0103: Attribute name "speedY" doesn't conform to snake_case naming style (invalid-name)
jogo.py:86:12: C0103: Variable name "player2Img" doesn't conform to snake_case naming style (invalid-name)
jogo.py:103:4: C0116: Missing function or method docstring (missing-function-docstring)
jogo.py:79:0: R0903: Too few public methods (1/2) (too-few-public-methods)
jogo.py:108:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:127:8: C0103: Attribute name "speedX" doesn't conform to snake_case naming style (invalid-name)
jogo.py:128:8: C0103: Attribute name "speedY" doesn't conform to snake_case naming style (invalid-name)
jogo.py:130:4: C0116: Missing function or method docstring (missing-function-docstring)
jogo.py:108:0: R0903: Too few public methods (1/2) (too-few-public-methods)
jogo.py:135:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:135:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:153:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:153:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:174:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:174:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:195:0: C0115: Missing class docstring (missing-class-docstring)
jogo.py:201:12: C0103: Variable name "EndScreen" doesn't conform to snake_case naming style (invalid-name)
jogo.py:195:0: R0903: Too few public methods (0/2) (too-few-public-methods)
jogo.py:211:0: C0116: Missing function or method docstring (missing-function-docstring)
jogo.py:211:0: R0914: Too many local variables (43/15) (too-many-locals)
jogo.py:215:4: C0103: Variable name "DISPLAY_X" doesn't conform to snake_case naming style (invalid-name)
jogo.py:216:4: C0103: Variable name "DISPLAY_Y" doesn't conform to snake_case naming style (invalid-name)
jogo.py:220:4: C0103: Variable name "backGround" doesn't conform to snake_case naming style (invalid-name)
jogo.py:222:4: C0103: Variable name "startScreen" doesn't conform to snake_case naming style (invalid-name)
jogo.py:224:4: C0103: Variable name "endScreen" doesn't conform to snake_case naming style (invalid-name)
jogo.py:231:4: C0103: Variable name "fontScore" doesn't conform to snake_case naming style (invalid-name)
jogo.py:232:4: C0103: Variable name "fontInstructions" doesn't conform to snake_case naming style (invalid-name)
jogo.py:237:4: C0103: Variable name "POS_Y" doesn't conform to snake_case naming style (invalid-name)
jogo.py:238:4: C0103: Variable name "POS_X" doesn't conform to snake_case naming style (invalid-name)
jogo.py:239:4: C0103: Variable name "DELTA_POS_X" doesn't conform to snake_case naming style (invalid-name)
jogo.py:240:4: C0103: Variable name "DELTA_POS_Y" doesn't conform to snake_case naming style (invalid-name)
jogo.py:243:4: C0103: Variable name "p1" doesn't conform to snake_case naming style (invalid-name)
jogo.py:244:4: C0103: Variable name "p2" doesn't conform to snake_case naming style (invalid-name)
jogo.py:247:4: C0103: Variable name "golEsq" doesn't conform to snake_case naming style (invalid-name)
jogo.py:248:4: C0103: Variable name "golDir" doesn't conform to snake_case naming style (invalid-name)
jogo.py:349:8: R1731: Consider using 'x = max(x, 0)' instead of unnecessary if block (consider-using-max-builtin)
jogo.py:388:8: C0103: Variable name "placarEsquerda" doesn't conform to snake_case naming style (invalid-name)
jogo.py:389:8: C0103: Variable name "placarDireita" doesn't conform to snake_case naming style (invalid-name)
jogo.py:472:12: C0103: Variable name "textoEsquerda" doesn't conform to snake_case naming style (invalid-name)
jogo.py:472:45: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
jogo.py:473:12: C0103: Variable name "textoDireita" doesn't conform to snake_case naming style (invalid-name)
jogo.py:473:44: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
jogo.py:481:16: C0103: Variable name "textoVencedor1" doesn't conform to snake_case naming style (invalid-name)
jogo.py:482:16: C0103: Variable name "textoInstrucoes1" doesn't conform to snake_case naming style (invalid-name)
jogo.py:483:16: C0103: Variable name "textoInstrucoes2" doesn't conform to snake_case naming style (invalid-name)
jogo.py:493:16: C0103: Variable name "textoVencedor1" doesn't conform to snake_case naming style (invalid-name)
jogo.py:494:16: C0103: Variable name "textoInstrucoes1" doesn't conform to snake_case naming style (invalid-name)
jogo.py:495:16: C0103: Variable name "textoInstrucoes2" doesn't conform to snake_case naming style (invalid-name)
jogo.py:211:0: R0912: Too many branches (15/12) (too-many-branches)
jogo.py:211:0: R0915: Too many statements (189/50) (too-many-statements)
jogo.py:234:4: W0612: Unused variable 'clock' (unused-variable)
jogo.py:4:0: C0411: standard import "import random" should be placed before "import pygame" (wrong-import-order)

------------------------------------------------------------------
Your code has been rated at 7.62/10 (previous run: 7.62/10, -0.00)


if anyone can help with the issues, I would be very gratefull!

--- a/game/jogo.py
+++ b/game/jogo.py
@@ -1,7 +1,12 @@
+# pylint: disable=too-few-public-methods
+"""
+Main game script
+"""
+
 import sys
 import os
-import pygame
 import random
+import pygame  # pylint: disable=import-error
 
 # Criando a gravidade
 GRAVITY = 3.5
@@ -15,31 +20,39 @@ WHITE = (255, 255, 255)
 YELLOW = (255, 255, 0)
 BLUE = (0, 0, 255)
 
+
 def resource_path(relative_path):
+    """ Get absolute path to resource, works for dev and for PyInstaller """
     try:
-    # PyInstaller creates a temp folder and stores path in _MEIPASS
-        base_path = sys._MEIPASS
-    except Exception:
+        # PyInstaller creates a temp folder and stores path in _MEIPASS
+        base_path = sys._MEIPASS  # pylint: disable=W0212
+    except AttributeError:
         base_path = os.path.abspath(".")
 
     return os.path.join(base_path, relative_path)
 
+
 def calcula_colisao_chao(sprite):
+    """ Compute collision with ground """
     if sprite.rect.bottom > 672:  # com o chao
         sprite.rect.bottom = 672
         sprite.speed_y = 0
 
-    sprite.rect.x = max(0,sprite.rect.x)
+    sprite.rect.x = max(0, sprite.rect.x)
     sprite.rect.x = min(1336 - sprite.rect.width, sprite.rect.x)
 
-class Background(pygame.sprite.Sprite):
+
+class Background(
+        pygame.sprite.Sprite):
+    """ Background class """
+
     def __init__(self):
 
         imagem = resource_path('Imagem/fundo.png')
         print(imagem)
         try:  # Importanto a imagem
             back_ground = pygame.image.load(imagem)  # da tela de fundo
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem de fundo")
             sys.exit()
 
@@ -48,14 +61,17 @@ class Background(pygame.sprite.Sprite):
         self.image = back_ground
 
 
-class Startscreen(pygame.sprite.Sprite):
+class Startscreen(
+        pygame.sprite.Sprite):
+    """ Start screen initialize """
+
     def __init__(self):
 
         imagem = resource_path('Imagem/startscreen.png')
         print(imagem)
         try:  # Importanto a imagem
             start_screen = pygame.image.load(imagem)  # da Tela Inicial
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem inicial")
             sys.exit()
 
@@ -65,19 +81,21 @@ class Startscreen(pygame.sprite.Sprite):
 
 
 class Jogador1(pygame.sprite.Sprite):
-    def __init__(self,column, row, block):
+    """ Player 1 initialize """
+
+    def __init__(self, column, row, block):
 
         imagem = resource_path('Imagem/ribamar.png')
         print(imagem)
         try:  # Importanto a imagem
-            player1Img = pygame.image.load(imagem)  # do jogador 1
-        except pygame.error:
+            player_1_img = pygame.image.load(imagem)  # do jogador 1
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem do jogador 1")
             sys.exit()
 
         pygame.sprite.Sprite.__init__(self)
 
-        self.image = player1Img
+        self.image = player_1_img
         self.blocks = block
         self.rect = self.image.get_rect()
         self.rect.x = column
@@ -88,24 +106,28 @@ class Jogador1(pygame.sprite.Sprite):
         self.speed_y = 0
 
     def update(self):
+        """ Updates player position """
+
         self.rect.x += self.speed_x
         self.rect.y += self.speed_y
 
 
 class Jogador2(pygame.sprite.Sprite):
+    """ Player 2 initialize """
+
     def __init__(self, colum, row, block):
 
         imagem = resource_path('Imagem/mece.png')
         print(imagem)
         try:  # Importanto a imagem
-            player2Img = pygame.image.load(imagem)  # do jogador 2
-        except pygame.error:
+            player_2_img = pygame.image.load(imagem)  # do jogador 2
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem do jogador 2")
             sys.exit()
 
         pygame.sprite.Sprite.__init__(self)
 
-        self.image = player2Img
+        self.image = player_2_img
         self.blocks = block
         self.rect = self.image.get_rect()
         self.rect.x = colum
@@ -116,18 +138,22 @@ class Jogador2(pygame.sprite.Sprite):
         self.speed_y = 0
 
     def update(self):
+        """ Updates player position """
+
         self.rect.x += self.speed_x
         self.rect.y += self.speed_y
 
 
 class Bola(pygame.sprite.Sprite):
+    """ Ball initialize """
+
     def __init__(self, colum, row, block):
 
         imagem = resource_path('Imagem/jabulani.png')
         print(imagem)
         try:  # Importanto a imagem
             ball = pygame.image.load(imagem)  # da bola
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar a bola")
             sys.exit()
 
@@ -143,18 +169,22 @@ class Bola(pygame.sprite.Sprite):
         self.speed_y = 0
 
     def update(self):
+        """ Updates ball position """
+
         self.rect.x += self.speed_x
         self.rect.y += self.speed_y
 
 
 class Campo(pygame.sprite.Sprite):
+    """ Field initialize """
+
     def __init__(self):
 
         imgfield = resource_path('Imagem/field.png')
         print(imgfield)
         try:  # Importando a imagem
             pitch = pygame.image.load(imgfield)  # do campo
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar o campo")
             sys.exit()
 
@@ -165,14 +195,17 @@ class Campo(pygame.sprite.Sprite):
         self.rect = self.image.get_rect()
 
 
-class GolEsquerdo(pygame.sprite.Sprite):
+class GolEsquerdo(
+        pygame.sprite.Sprite):
+    """ GolEsquerdo initialize """
+
     def __init__(self):
 
         imggol1 = resource_path('Imagem/gol-esq.png')
         print(imggol1)
         try:  # Importando a imagem
             g_esq = pygame.image.load(imggol1)  # do gol esquerdo
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem do gol esquerdo")
             sys.exit()
 
@@ -185,14 +218,17 @@ class GolEsquerdo(pygame.sprite.Sprite):
         self.mask = pygame.mask.from_surface(self.image)
 
 
-class GolDireito(pygame.sprite.Sprite):
+class GolDireito(
+        pygame.sprite.Sprite):
+    """ GolDireito initialize """
+
     def __init__(self):
 
         imggol2 = resource_path('Imagem/gol-dir.png')
         print(imggol2)
         try:  # Impotando a imagem
             g_dir = pygame.image.load(imggol2)  # do gol direito
-        except pygame.error:
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem do gol direito")
             sys.exit()
 
@@ -205,23 +241,28 @@ class GolDireito(pygame.sprite.Sprite):
         self.rect.bottom = 672
 
 
-class End_screen(pygame.sprite.Sprite):
+class EndScreen(
+        pygame.sprite.Sprite):
+    """ End screen initialize """
+
     def __init__(self):
 
         imagem = resource_path('Imagem/endscreen.png')
         print(imagem)
         try:  # Importanto a imagem
-            End_screen = pygame.image.load(imagem)  # da end_screen
-        except pygame.error:
+            end_screen = pygame.image.load(imagem)  # da end_screen
+        except pygame.error:  # pylint: disable=E1101
             print("Erro ao carregar imagem final")
             sys.exit()
 
         pygame.sprite.Sprite.__init__(self)
 
-        self.image = End_screen
+        self.image = end_screen
 
 
-def main():  # main routine
+def main():  # pylint: disable=R0914 disable=R0912 disable=R0915
+    """ Main routine """
+
     pygame.init()
 
     # dimensoes do display, em X e Y
@@ -234,7 +275,7 @@ def main():  # main routine
 
     start_screen = Startscreen()
 
-    end_screen = End_screen()
+    end_screen = EndScreen()
 
     # Criando o booleano para configurar a tela inicial:
     start = True
@@ -243,7 +284,6 @@ def main():  # main routine
 
     font_score = pygame.font.Font(pygame.font.get_default_font(), 35)
     font_instructions = pygame.font.Font(pygame.font.get_default_font(), 20)
-
 
     # n tem contstante no python, mas letra maiuscula indica constante
     pos_y = int(display_y / 4)
@@ -256,26 +296,26 @@ def main():  # main routine
     p_2 = Jogador2(1336 - 350, pos_y, 'block')
     jabulani = Bola(display_x / 2, display_x / 2, 'block')
     cancha = Campo()
-    golEsq = GolEsquerdo()
-    golDir = GolDireito()
+    gol_esq = GolEsquerdo()
+    gol_dir = GolDireito()
 
     sprites = pygame.sprite.Group()  # criando o grupo de sprites
     sprites.add(p_1, p_2, jabulani)  # adiciona as sprites ao grupo de sprites
 
     # inicializa a musica de fundo
-    torcida = resource_path('Som/Torcida.ogg')# Som ambiente de torcida
-    # torcida = os.path.join('Som', 'Torcida.ogg')  
+    torcida = resource_path('Som/Torcida.ogg')  # Som ambiente de torcida
+    # torcida = os.path.join('Som', 'Torcida.ogg')
     pygame.mixer.music.load(torcida)  # Carrega som
     pygame.mixer.music.set_volume(0.2)  # Volume
     pygame.mixer.music.play(-1)  # Toca som
 
     # inicia os outros sons
 
-    sompulo= resource_path('Som/Pular.ogg') # som do pulo
+    sompulo = resource_path('Som/Pular.ogg')  # som do pulo
     pulo = pygame.mixer.Sound(sompulo)  # carrega som
-    somchute = resource_path('Som/Chute.ogg')# som do chute
+    somchute = resource_path('Som/Chute.ogg')  # som do chute
     chute = pygame.mixer.Sound(somchute)  # carrega som
-    somgol = resource_path('Som/Gol.ogg')# som do gol
+    somgol = resource_path('Som/Gol.ogg')  # som do gol
     gol = pygame.mixer.Sound(somgol)  # carrega som
     gol.set_volume(0.7)
 
@@ -331,32 +371,33 @@ def main():  # main routine
             pulo.play()
 
     def calcula_colisao_p_bola(player1, player2, bola):
-        if pygame.sprite.collide_mask(player1, bola):  
+        if pygame.sprite.collide_mask(player1, bola):
             # player 1 com  a bola, garantido que ele soh possa
             # chutar para o gol do p_2
             if player1.speed_x >= 0:
                 bola.speed_x = player1.speed_x * (random.uniform(1, 3))
-                bola.speed_y = (-1) * random.randint(12, 18) + player1.speed_y * (random.uniform(0.1, 0.7))
+                bola.speed_y = (-1) * random.randint(12, 18) + \
+                    player1.speed_y * (random.uniform(0.1, 0.7))
                 chute.play()
             else:
                 bola.speed_x = player1.speed_x * (random.uniform(1, 3)) * (-1)
                 bola.speed_y = -player1.speed_y * (random.uniform(2, 4))
                 chute.play()
 
-        if pygame.sprite.collide_mask(player2, bola):  
+        if pygame.sprite.collide_mask(player2, bola):
             # player 2 com  a bola, garantido que ele soh possa
             # chutar para o gol do p_1
 
             if player2.speed_x <= 0:
                 bola.speed_x = player2.speed_x * (random.uniform(1, 3))
-                bola.speed_y = (-1) * random.randint(12, 18) + player2.speed_y * (random.uniform(0.1, 0.7))
+                bola.speed_y = (-1) * random.randint(12, 18) + \
+                    player2.speed_y * (random.uniform(0.1, 0.7))
                 chute.play()
 
             else:
                 bola.speed_x = player2.speed_x * (random.uniform(1, 3)) * (-1)
                 bola.speed_y = -player2.speed_y * (random.uniform(2, 4))
                 chute.play()
-
 
     def calcula_colisao_gol(gol_dir, gol_esq, bola):
         if pygame.sprite.collide_rect(bola, gol_dir):
@@ -407,14 +448,16 @@ def main():  # main routine
                 start = False
 
             # reincia o jogo caso queiram jogar denovo apos
-            if event.type == pygame.KEYDOWN and event.key == pygame.K_RETURN and (p_1.score >= 7 or p_2.score >= 7):
+            if event.type == pygame.KEYDOWN and event.key == pygame.K_RETURN and (
+                    p_1.score >= 7 or p_2.score >= 7):
                 p_1.score = 0
                 p_2.score = 0
 
                 reset()
 
             # quitando o jogo, fora da tela inicial
-            if event.type == pygame.QUIT or event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE:
+            if event.type == pygame.QUIT or event.type == pygame.KEYDOWN \
+                    and event.key == pygame.K_ESCAPE:
                 pygame.quit()
                 sys.exit()
 
@@ -440,10 +483,13 @@ def main():  # main routine
                 # colisao player com bola
                 calcula_colisao_p_bola(p_1, p_2, jabulani)
 
-                # colisões com o gol, toca o som, aumenta o placar, e reseta todos os sprites para a posicao inicial,
+                # colisões com o gol,
+                # toca o som,
+                # aumenta o placar,
+                # e reseta todos os sprites para a posicao inicial,
                 # com, velocidade X e Y = 0
 
-                calcula_colisao_gol(golDir, golEsq, jabulani)
+                calcula_colisao_gol(gol_dir, gol_esq, jabulani)
 
                 # gravidade, e atrito na bola
                 jabulani.speed_y += GRAVITY / 20
@@ -471,12 +517,14 @@ def main():  # main routine
             surf.blit(p_2.image, p_2.rect)  # player 2
             surf.blit(jabulani.image, jabulani.rect)  # bola
             surf.blit(cancha.image, [0, 672])  # campo
-            surf.blit(golEsq.image, [0, 320])  # gol da esquerda
-            surf.blit(golDir.image, [1177, 320])  # gol da direita
+            surf.blit(gol_esq.image, [0, 320])  # gol da esquerda
+            surf.blit(gol_dir.image, [1177, 320])  # gol da direita
 
             # Adicionando os placares:
-            texto_esquerda = font_score.render("Ribamar: {0}".format(placar_esquerda), True, YELLOW)
-            texto_direita = font_score.render("Messi Careca: {0}".format(placar_direita), True, YELLOW)
+            texto_esquerda = font_score.render(
+                f"Ribamar: {placar_esquerda}", True, YELLOW)
+            texto_direita = font_score.render(
+                f"Messi Careca: {placar_direita}", True, YELLOW)
             surf.blit(texto_esquerda, (10, 0))
             surf.blit(texto_direita, (1045, 0))
 
@@ -484,26 +532,38 @@ def main():  # main routine
             if p_1.score >= 7:
                 surf.fill(BLACK)
                 surf.blit(end_screen.image, [display_x / 5, display_x / 5])
-                texto_vencedor_1 = font_instructions.render("Jogador 1 venceu!", True, WHITE)
-                texto_instrucoes_1 = font_instructions.render("Pressione 'Enter' para jogar novamente", True, WHITE)
-                texto_instrucoes_2 = font_instructions.render("Pressione 'ESC' para sair", True, WHITE)
+                texto_vencedor_1 = font_instructions.render(
+                    "Jogador 1 venceu!", True, WHITE)
+                texto_instrucoes_1 = font_instructions.render(
+                    "Pressione 'Enter' para jogar novamente", True, WHITE)
+                texto_instrucoes_2 = font_instructions.render(
+                    "Pressione 'ESC' para sair", True, WHITE)
                 surf.blit(p_1.image, (display_x * 0.47, display_x / 3))
                 surf.blit(texto_vencedor_1, [display_x * 4 / 9, display_x / 3])
-                surf.blit(texto_instrucoes_1, [display_x * 4 / 11, display_x * 3 / 5])
-                surf.blit(texto_instrucoes_2,
-                          [(display_x / 2) - (texto_instrucoes_2.get_rect().width / 2), display_x * 11 / 17])
+                surf.blit(
+                    texto_instrucoes_1, [
+                        display_x * 4 / 11, display_x * 3 / 5])
+                surf.blit(texto_instrucoes_2, [
+                    (display_x / 2) - (texto_instrucoes_2.get_rect().width / 2),
+                    display_x * 11 / 17])
 
             if p_2.score >= 7:
                 surf.fill(BLACK)
                 surf.blit(end_screen.image, [display_x / 5, display_x / 5])
-                texto_vencedor_1 = font_instructions.render("Jogador 2 venceu!", True, WHITE)
-                texto_instrucoes_1 = font_instructions.render("Pressione 'Enter' para jogar novamente", True, WHITE)
-                texto_instrucoes_2 = font_instructions.render("Pressione 'ESC' para sair", True, WHITE)
+                texto_vencedor_1 = font_instructions.render(
+                    "Jogador 2 venceu!", True, WHITE)
+                texto_instrucoes_1 = font_instructions.render(
+                    "Pressione 'Enter' para jogar novamente", True, WHITE)
+                texto_instrucoes_2 = font_instructions.render(
+                    "Pressione 'ESC' para sair", True, WHITE)
                 surf.blit(p_2.image, (display_x * 0.47, display_x / 3))
                 surf.blit(texto_vencedor_1, [display_x * 4 / 9, display_x / 3])
-                surf.blit(texto_instrucoes_1, [display_x * 4 / 11, display_x * 3 / 5])
-                surf.blit(texto_instrucoes_2,
-                          [(display_x / 2) - (texto_instrucoes_2.get_rect().width / 2), display_x * 11 / 17])
+                surf.blit(
+                    texto_instrucoes_1, [
+                        display_x * 4 / 11, display_x * 3 / 5])
+                surf.blit(texto_instrucoes_2, [
+                    (display_x / 2) - (texto_instrucoes_2.get_rect().width / 2),
+                    display_x * 11 / 17])
 
         pygame.display.flip()  # faz o update da imagine, usando troca de memory bugger
 

